content/en/tracing/connect_logs_and_traces/python.md

```diff
@@ -76,18 +70,9 @@ import structlog
 
 def tracer_injection(logger, log_method, event_dict):
     # get correlation ids from current tracer context
-    span = tracer.current_span()
-    trace_id, span_id = (span.trace_id, span.span_id) if span else (None, None)
-
-    # add ids to structlog event dictionary
-    event_dict['dd.trace_id'] = str(trace_id or 0)
-    event_dict['dd.span_id'] = str(span_id or 0)
-
-    # add the env, service, and version configured for the tracer
-    event_dict['dd.env'] = ddtrace.config.env or ""
-    event_dict['dd.service'] = ddtrace.config.service or ""
-    event_dict['dd.version'] = ddtrace.config.version or ""
-
+    log_correlation_context = tracer.get_log_correlation_context()
+    
+    event_dict["dd"] = log_correlation_context
```

Comment:
```suggestion
    event_dict["dd"] = tracer.get_log_correlation_context()
```

Think we can simplify a little further.

We can probably update the comment above this line, maybe something like:

> add current trace context to the log event


? (soft suggestion, not sure I _love_ it, but... someplace to start)